test/testlock.c

```diff
@@ -102,7 +102,7 @@ Run(void *data)
 #ifndef _WIN32
 static Uint32 hit_timeout(Uint32 interval, void *param) {
     SDL_Log("Hit timeout! Sending SIGINT!");
-    kill(0, SIGINT);
+    (void)raise(SIGINT);
```

Comment:
`(void)` is needed here but it wasn't needed for kill() ??

Reply:
Somehow clang-tidy just shows a warning for the `kill()` function and not for `raise()`.
And it also is not in the list of checked functions:
[https://clang.llvm.org/extra/clang-tidy/checks/cert/err33-c.html](url)